io/EptReader.cpp

```diff
@@ -772,11 +784,10 @@ PointId EptReader::readLaszip(PointView& dst, const Key& key,
     return startId;
 }
 
-PointId EptReader::readBinary(PointView& dst, const Key& key,
-        const uint64_t nodeId) const
+PointId EptReader::processPackedData(PointView& dst, const uint64_t nodeId,
+    char* data, const uint64_t size) const
 {
-    auto data(getBinary("ept-data/" + key.toString() + ".bin"));
-    ShallowPointTable table(*m_remoteLayout, data.data(), data.size());
+    ShallowPointTable table(*m_remoteLayout, data, size);
     PointRef pr(table);
 
     std::lock_guard<std::mutex> lock(m_mutex);
```

Comment:
There's an extra `pointId` declaration below.  Can't figure out how to mark that line.

Reply:
Good catch.  d3dc6344d5bf4bf3a90616f5ac13d107fab68c97